app/react/package.json

```diff
@@ -18,7 +18,8 @@
     "url": "https://github.com/storybooks/storybook.git"
   },
   "scripts": {
-    "dev": "DEV_BUILD=1 nodemon --watch ./src --exec 'yarn prepare'",
+    "prepare-root": "cd ../../ && yarn && cd ./app/react",
```

Comment:
We use [yarn workspaces](https://yarnpkg.com/en/docs/workspaces), so you can run `yarn install` in any package directory, and it will bootstrap everything. The `cd` part is therefore redundant

Reply:
Oh that's curious. It looks like Yarn just released 1.2.1, perhaps this was a bug fixed with workspaces.

Reply:
Would you like me to remove the `prepare-root` script from this and retry from a clean git pull?

Reply:
Yes please, but you can actually do `yarn bootstrap --reset` instead of clean pull

Reply:
Hey, I commented further below, could you check in on this?